src/initialize_dae.jl

```diff
@@ -20,13 +20,12 @@ function BrownFullBasicInit(; abstol = 1e-10, nlsolve = nothing)
 end
 BrownFullBasicInit(abstol) = BrownFullBasicInit(; abstol = abstol, nlsolve = nothing)
 
-using SciMLNLSolve
 default_nlsolve(alg, isinplace, u, autodiff = false) = alg
 function default_nlsolve(::Nothing, isinplace, u, autodiff = false)
-    NLSolveJL(autodiff = autodiff ? :forward : :central)
+    TrustRegion(; autodiff = autodiff ? AutoForwardDiff() : AutoFiniteDiff())
 end
 function default_nlsolve(::Nothing, isinplace::Val{false}, u::StaticArray, autodiff = false)
-    SimpleNewtonRaphson(autodiff = autodiff)
+    SimpleNewtonRaphson(autodiff = autodiff ? AutoForwardDiff() : AutoFiniteDiff())
```

Comment:
@ChrisRackauckas should we change this to `SimpleTrustRegion` to make it consistent with the non static array version?

Reply:
Yes